stripes/www/stripes_home.html

```diff
@@ -227,14 +227,47 @@ <h1 class="m-0 logo">Pollution Stripes</h2>
 			}
 		};
 
+		/* Fetching Stirpes */
+		const fetch_stripes = (value) => {
+			const stripes_container = document.querySelector(".stripes-container");
+			stripes_container?.classList.add("loading");
+
+			frappe.call({
+				method: 'stripes.svg.get_stripes_svg',
+				args: {
+					from_date: "2018-01-01",
+					to_date: "2018-12-31",
+					monitor_region: value
+				},
+				callback: (response) => {
+					if (response.message) {
+						document.querySelector(".stripes-container .stripes").innerHTML = response.message;
+					}
+					stripes_container?.classList.remove("loading");
+				},
+				error: (error) => {
+					stripes_container?.classList.remove("loading");
+				}
```

Comment:
Maybe lets not remove the loader .. just frappe.msgprint or something that an error occurred

Reply:
Wo to wese bhi araha hai.
![Uploading image.png…]()

Reply:
Whenever I didn't remove the loader before the error.

Reply:
This error is something else, the error seems like a bug from your side rather than missing data